components/databox/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@pipedream/databox",
+  "version": "0.0.2",
```

Shoulnd't you start with?
```suggestion
  "version": "0.0.1",
```